charts/umbrella/README.md

```diff
@@ -283,7 +283,7 @@ helm delete umbrella --namespace umbrella
 
 ## How to contribute
 
-Before contributing, make sure, you read and understood our [contributing guidelines](./CONTRIBUTING.md).
+Before contributing, make sure, you read and understand our [contributing guidelines](./CONTRIBUTING.md).
```

Comment:
if you'r already at it: could you please also fix the reference to the CONTRIBUTING.md file at root level, it's broken

Reply:
Sure thing, I fixed it.